style.css

```diff
@@ -0,0 +1,816 @@
+  color: #a0a0a0;
+  text-decoration: none;
+}
+
+h6 > a:hover {
+  color: black;
+  text-decoration: none;
+  border-bottom: 1px solid red;
+}
+
+.nav_bar_mid > li {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  width: 100%;
+  border-left: 1px solid #d9d9d9;
+}
+
+.nav_bar_mid > li > a {
+  color: black;
+  font-weight: bold;
+}
+
+.search_part {
+  gap: 10px;
+}
+
+#border_red {
+  border-top: 20px solid red;
+  border-left: 20px solid #f4f5f6;
+}
+
+.last_nav {
+  height: 46px;
+  width: 100%;
+  border-top: 1px solid #f4f5f7;
+  border-bottom: 1px solid #f4f5f7;
+}
+
+.first_button {
+  font-size: 12px;
+  height: 25px;
+  border-radius: 24px;
+  background-color: #ffe400;
+}
+
+.div_gap1 {
+  column-gap: 20px;
+}
+
+.div_gap2 {
+  column-gap: 20px;
+}
+
+.title_sec {
+  background-color: #f61602;
+  font-size: 20px;
+  color: white;
+  padding: 5px 8px 5px 8px;
+}
+
+.editor_title {
+  padding: 5px;
+  font-size: 20px;
+}
+
+.in_images {
+  background-image: url(https://d.newsweek.com/en/full/1701186/ph3095-01.jpg?w=500&h=333&f=b8162daf93e087918df15d58a79bfc35);
+  height: 200px;
+}
+
+.title_form {
+  background-color: #f72210;
+  padding: 5px 9px 5px 9px;
+  color: white;
+}
+
+.nav_bar_bot {
+  height: 100%;
+  width: 100%;
+}
+
+.nav_bar_bot > li {
+  margin: auto;
+}
+
+.nav_bar_bot > li > a {
+  color: #807f84;
+}
+
+.text_deco > li > a {
+  color: black;
+  text-decoration: none;
+}
+
+.nav_bot2 > ul > li > a {
+  font-size: 14px;
+  color: #666;
+}
+
+.border_top > div > a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+.nav_bar_mid > li > a:hover {
+  color: red;
+  text-decoration: none;
+}
+
+.nav_bar_bot > li > a:hover {
+  color: red;
+  text-decoration: none;
+}
+
+/* phone navbar */
+
+#phone_navbar {
+  background-color: #f72210;
+  width: 100%;
+  height: 50px;
+  position: sticky;
+  top: 0;
+  z-index: 3;
+}
+
+#phone_cont {
+  width: 95%;
+}
+
+.part1 > article {
+  border-top: 1px solid #ddd;
+  padding-top: 10px;
+}
+
+.part2 > article {
+  border-top: 1px solid #ddd;
+  padding-top: 10px;
+}
+
+.magazin_cont > div > div {
+  gap: 10px;
+  padding: 15px 0 0 0;
+}
+
+.editor_cont > div > div {
+  gap: 10px;
+  padding: 15px 0 0 0;
+}
+
+#phone_cont > div {
+  display: flex;
+  align-items: center;
+  gap: 25px;
+  cursor: pointer;
+}
+
+#phone_navbar > div > img {
+  cursor: pointer;
+  width: 132px;
+  height: 30px;
+}
+
+#user {
+  color: #fdc7c3;
+}
+
+#menu {
+  color: #fb9b93;
+  font-size: larger;
+}
+
+/* phone main */
+
+main {
+  background-color: #f4f5f6;
+  height: auto;
+  margin-top: 20px;
+}
+
+.section_cont {
+  background-color: white;
+}
+
+.featured {
+  row-gap: 15px;
+}
+
+.topstory {
+  row-gap: 15px;
+}
+
+.background1 {
+  background-image: url(https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1711616/donald-trump.webp?w=397&amp;h=265&amp;q=90&amp;f=d759057143d283aa0ea0ed13fba5450a);
+  background-size: cover;
+  background-position: center;
```

Comment:
Bootstrap has classes for these as well.

## Spare enough time and search for all classes you can use for this project. Minimize the number of CSS lines you have.